tests/functional/conftest.py

```diff
@@ -129,12 +129,12 @@ def increase_pps(vault, token, amount, sender):
         token.transfer(strategy, amount, {"from": sender})
         managementFee = vault.managementFee()
         performanceFee = vault.performanceFee()
-        vault.setManagementFee(0)
-        vault.setPerformanceFee(0)
-        vault.updateStrategyPerformanceFee(strategy, 0)
-        strategy.harvest()
-        vault.setManagementFee(managementFee)
-        vault.setPerformanceFee(performanceFee)
+        vault.setManagementFee(0, {"from": vault.governance()})
```

Comment:
You shouldn't change unnecessary parts of the test.

Reply:
After I moved `strategy` in the argument list, when I use it the test, it's reverting because `msg.sender != vault.governance()`, so I explicitly set "from" to `vault.governance()`.

Reply:
If I apply this commit on main branch, it reverts for me: https://github.com/gmridul/yearn-vaults/commit/3b04e62651e4c29ffbbeec61c10385d0973688d5

```
brownie test tests/functional/vault/test_withdrawal.py -k test_withdrawal_with_empty_queue --interactive
```

In any case, I think it's safe to explicitly specify `msg.sender` for these calls.

Reply:
Thanks for the clean up :)